src/todolist/settings.py

```
@@ -18,7 +18,7 @@
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*"
+SECRET_KEY = os.environ["SECRET_KEY"]
```

The code correctly reads SECRET_KEY from the environment which satisfies the task requirement (checklist item that settings.py must use the secret instead of a hardcoded value). However, `SECRET_KEY = os.environ["SECRET_KEY"]` will raise a KeyError if the env var is absent. Consider either documenting that the Secret must be created before the Deployment in INSTRUCTION.md, or making the check explicit to produce a clearer error message. Example alternatives:

- Keep fail-fast but document ordering: ensure `kubectl create -f .infrastructure/secret.yml` (and namespace) before applying the Deployment.
- Or change to a clearer guard:
  ```python
  SECRET_KEY = os.environ.get("SECRET_KEY")
  if not SECRET_KEY:
      raise RuntimeError("Missing SECRET_KEY environment variable. Create the todo-secret in namespace todoapp before starting the app.")
  ```

This is optional for task correctness but will improve debugging if the Secret or namespace is not applied correctly.